pyverbs/qp.pyx

```diff
@@ -902,14 +902,17 @@ cdef class QP(PyverbsCM):
                     xrcd = <XRCD>init_attr.xrcd
                     xrcd.add_ref(self)
                     self.xrcd = xrcd
+                if self.qp == NULL:
+                    raise PyverbsRDMAErrno('Failed to create extended QP')
```

Comment:
I don't think this is correct - we're still creating ibv_qp and not ibv_qp_ex.

Reply:
Will that because the ibv_create_qp() and ibv_create_qp_ex() functions reutnrn "ibv_qp *".

Reply:
I know, this makes your message inaccurate. Please change that.

Reply:
OK, I'll drop this patch.

Reply:
It seems reasonable to throw when a create fails, but shouldn't this be right after the system call, why are we doing refcounting for something that doesn't need refcounting?

Reply:
OK, I'll change this patch to raise the exception right after create QP fail.